src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -446,20 +447,38 @@ public void initializeFromEntity(@NotNull final EntityCitizen entity)
     private String generateName(@NotNull final Random rand)
     {
         String citizenName;
+        String firstName;
+        String middleInitial;
+        String lastName;
+
         if (female)
         {
-            citizenName = String.format("%s %s. %s", getRandomElement(rand, Configurations.names.femaleFirstNames), getRandomLetter(rand),
-              getRandomElement(rand, Configurations.names.lastNames));
+            firstName = getRandomElement(rand, Configurations.names.femaleFirstNames);
+        }
+        else
+        {
+            firstName = getRandomElement(rand, Configurations.names.maleFirstNames);
+        }
+
+        middleInitial = String.valueOf(getRandomLetter(rand));
+        lastName = getRandomElement(rand, Configurations.names.lastNames);
+
+        if(Configurations.names.useMiddleInitial) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)

Reply:
only that and we can merge this =)